lib/avo/resources/base.rb

```diff
@@ -359,7 +361,7 @@ def divider(label = nil)
       end
 
       def hydrate(...)
-        super(...)
+        super
```

Comment:
Does this work with Ruby 3.1?

Reply:
Super always passes every argument to the parent method. It's always been that way, I think. Paul also noticed this and had asked: https://github.com/avo-hq/avo/pull/3048#discussion_r1698293700

Reply:
I tested this on ruby:
```
(ins)$ ruby -v
ruby 3.0.3p157 (2021-11-24 revision 3fb7d2cadc) [x86_64-linux]
```
and it works!
```ruby
class Parent
  def hydrate(**args)
    puts args
  end
end

class Child < Parent
  def hydrate(...)
    super
  end
end

Child.new.hydrate(tim: 'cook', steve: 'jobs')
#=> {:tim=>"cook", :steve=>"jobs"}
```

This other article I found says that `...` is exactly like calling super without any arguments: https://blog.saeloun.com/2019/12/04/ruby-2-7-adds-new-operator-for-arguments-forwarding/

![image](https://github.com/user-attachments/assets/ad888035-ffa1-4696-9548-de4914235a5a)